lib/streamlit/DeltaGenerator.py

```diff
@@ -2126,6 +2131,15 @@ def text_input(self, element, label, value="", key=None):
         """
         element.text_input.label = label
         element.text_input.default = str(value)
+        if type == "default":
+            element.text_input.type = TextInput.DEFAULT
+        elif type == "password":
+            element.text_input.type = TextInput.PASSWORD
+        else:
+            raise StreamlitAPIException(
+                "'%s' is not a valid text_input type. Valid types are 'default' and 'password'."
+                % str(type)
```

Comment:
Just curious why you need the `str()` call here, it wasn't obvious to me.

Reply:
I don't think it's necessary at all! I'll remove it, thanks for the catch.